app/assets/stylesheets/components/_banner.scss

```diff
@@ -48,7 +39,11 @@
 }
 
 .app-c-banner__desc {
-  @include core-24;
+  @include core-19;
   // Ensure the text has a line-length of around 60 characters
   max-width: 30em;
 }
+
+.consultation-date {
```

Comment:
When I created the banner component, it was suggested that the consultation date styling should still live with the consultation CSS scope as it's not technically a class within this component (it's not scoped to the banner CSS - app-c-banner etc). 
https://github.com/alphagov/government-frontend/pull/424#discussion_r130863515

Reply:
I'm not 100% sure what the technical impact of this change is, so we can drop it if its bad practice.

Reply:
@vanitabarrett should we move this code to consultations

Reply:
@alextea Yeah, should be in `consultations.scss`. This PR doesn't actually delete it from that file, so it might just be a case of deleting in the component CSS and checking it still exists in `consultations.scss`